src/client.rs

```diff
@@ -2,8 +2,29 @@ use std::cell::RefCell;
 use std::time::Duration;
 
 use cached::Cached;
+#[cfg(any(
+    feature = "account_balance",
+    feature = "b2b",
+    feature = "b2c",
+    feature = "transaction_reversal",
+    feature = "transaction_status"
+))]
 use openssl::base64;
+#[cfg(any(
+    feature = "account_balance",
+    feature = "b2b",
+    feature = "b2c",
+    feature = "transaction_reversal",
+    feature = "transaction_status"
+))]
 use openssl::rsa::Padding;
+#[cfg(any(
+    feature = "account_balance",
+    feature = "b2b",
+    feature = "b2c",
+    feature = "transaction_reversal",
+    feature = "transaction_status"
+))]
 use openssl::x509::X509;
```

Comment:
This could be possible cleaned up by deriving a `feature_locked!`  (better name needed) declarative macro